float32 x
float32 y
float32 straight_line
float32 turn_angle
float32 current_bearing
float32 UTM_x
float32 UTM_y
